python/py-slackclient/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        slackapi python-slackclient 2.7.2 v
+name                py-slackclient
+categories-append   irc
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     36 37 38
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         Slack Web API and Real Time Messaging (RTM) API.
+long_description    The Python slackclient is a developer kit for interfacing with the Slack \
+                    Web API and Real Time Messaging (RTM) API on Python 3.6 and above.
+
+homepage            https://slack.dev/python-slackclient/
+
+checksums           rmd160  0ad05eb4c7ace9b5228a8dbe8fa520defa4b8a15 \
+                    sha256  0908181db3448fb5d14e67d931afc341c6a73d18d26ca791c512f33fe85fa748 \
+                    size    2461167
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+    depends_lib-append      port:py${python.version}-aiohttp
```

Comment:
please add `livecheck.type  none` within the ` if {${name} ne ${subport}} {` block - we typically don't do livecheck for the subports, but only for `py-<portname>`